$NetBSD$
Support exception for NetBSD, taken from FreeBSD. Correctly locate the library
--- config.mk.orig	2016-06-02 21:20:22.000000000 +0000
+++ config.mk	2016-08-28 15:27:03.968104814 +0000
@@ -115,7 +115,7 @@
 BROKER_CFLAGS:=${LIB_CFLAGS} ${CPPFLAGS} -DVERSION="\"${VERSION}\"" -DTIMESTAMP="\"${TIMESTAMP}\"" -DWITH_BROKER
 CLIENT_CFLAGS:=${CFLAGS} ${CPPFLAGS} -I../lib -DVERSION="\"${VERSION}\""
 
-ifneq ($(or $(findstring $(UNAME),FreeBSD), $(findstring $(UNAME),OpenBSD)),)
+ifneq ($(or $(findstring $(UNAME),FreeBSD), $(findstring $(UNAME),OpenBSD), $(findstring $(UNAME),NetBSD)),)
 	BROKER_LIBS:=-lm
 else
 	BROKER_LIBS:=-ldl -lm
@@ -128,7 +128,7 @@
 	LIB_LIBS:=$(LIB_LIBS) -lrt
 endif
 
-CLIENT_LDFLAGS:=$(LDFLAGS) -L../lib ../lib/libmosquitto.so.${SOVERSION}
+CLIENT_LDFLAGS:=$(LDFLAGS) libmosquitto.so.${SOVERSION}
 
 ifeq ($(UNAME),SunOS)
 	ifeq ($(CC),cc)
@@ -241,7 +241,7 @@
 endif
 
 INSTALL?=install
-prefix=/usr/local
+prefix=${PREFIX}
-mandir=${prefix}/share/man
+mandir=${prefix}/man
 localedir=${prefix}/share/locale
 STRIP?=strip
